nix/overlays.nix

```diff
@@ -76,18 +94,26 @@ in {
       value = recurseIntoAttrs (builtins.listToAttrs (builtins.map
         (llvmVersion: {
           name = "llvm-" + toString llvmVersion;
-          value = recurseIntoAttrs ({
-            hobbes = dbg
-              (callPackage (withGCC { inherit (gccConstraint) gccVersion; }) {
-                inherit llvmVersion;
+          value = recurseIntoAttrs (builtins.listToAttrs (builtins.map
+            (sans: {
+              name = (if sans then "ASanAndUBSan" else "Default");
+              value = recurseIntoAttrs ({
+                hobbes = dbg
+                  (callPackage (withGCC { inherit sans; inherit (gccConstraint) gccVersion; }) {
+                    inherit llvmVersion;
+                  });
               });
-          });
+            }) asanAndUBSan));
         }) gccConstraint.llvmVersions));
     }) gccConstraints))) // recurseIntoAttrs (builtins.listToAttrs (builtins.map
       (llvmVersion: {
         name = "clang-" + toString llvmVersion;
-        value = recurseIntoAttrs ({
-          hobbes = dbg (callPackage withCLANG { inherit llvmVersion; });
-        });
+        value = recurseIntoAttrs (builtins.listToAttrs (builtins.map
```

Comment:
Add new separate SAN specific packages `withSans` instead. Do not add params to the existing ones